manifests/identity_wincred.pp

```diff
@@ -2,7 +2,7 @@
   if $conjur::api_key {
 
     # Temporarily use the registry as a stand-in for WinCred
-    $client_url = $conjur::client[uri]
+    $client_url = URI($conjur::client[uri]) + 'authn'
```

Comment:
Is there a good reason for this?

Reply:
Do you mean the `authn` suffix?

If so, I was following what I saw here: https://github.com/cyberark/conjur-puppet/blob/4aae7d6e54310cc402ddbdd87150f509c11a7b54/lib/puppet/functions/conjur/netrc.rb#L13

AFAICT, it's not necessary.

Reply:
Ah, this is because some old clients use and expect full authn url in netrc file in 'host' field. Strictly speaking this should just be a hostname (I think this is also true for wincred), but we're kind of stuck with this until we're sure there aren't any that don't match for a naked hostname (which might already be true). Since we don't have any legacy clients using wincred we might as well do it correctly from the start :)

Reply:
Sounds good. I'll switch to the hostname alone.

Reply:
Thanks!